sympy/integrals/integrals.py

```diff
@@ -525,8 +525,8 @@ def try_meijerg(function, xab):
                     terms = tuple((antideriv,))
                 for term in terms:
                     atan_part = term.atoms(atan)
-                    if atan_part and len(atan_part)==1:
-                        atan_arg = atan_part.pop().args[0]
+                    for atan_term in atan_part:
+                        atan_arg = atan_term.pop().args[0]
```

Comment:
`atan_term` is an element of `atan_part`, hence no `pop` is needed.

Reply:
oh yes sorry!!